RDF/StaticAnalysis.hs

```diff
@@ -35,31 +35,35 @@ import Common.Lib.State
 -- * URI Resolution
 
 resolveFullIRI :: IRI -> IRI -> IRI
-resolveFullIRI absol rel = if isAbsoluteIRI rel then rel else
-    let r = fromJust $ IRI.parseIRIReference $ expandedIRI rel
-        a = fromJust $ IRI.parseIRI $ expandedIRI absol
+resolveFullIRI _absol rel = rel
+ {-if isAbsoluteIRI rel then rel else
+    let r = fromJust $ parseIRIReference $ fromJust $ expandCurie Map.empty rel
+        a = fromJust $ parseIRI $ fromJust $ expandCurie Map.empty absol
         Just ra = IRI.relativeTo r a
         resolved = IRI.iriToStringUnsecure ra
         Right new = parse uriQ "" $ "<" ++ resolved ++ ">"
     in rel {expandedIRI = expandedIRI new}
+ -}
 
 resolveAbbreviatedIRI :: RDFPrefixMap -> IRI -> IRI
-resolveAbbreviatedIRI pm new = case Map.lookup (namePrefix new) pm of
+resolveAbbreviatedIRI pm new = fromJust $ expandCurie pm new
+  {-case Map.lookup (namePrefix new) pm of
     Nothing -> error $ namePrefix new ++ ": prefix not declared"
-    Just iri -> let new2 = if null (namePrefix new)
+    Just anIRI -> let new2 = if null (namePrefix new)
```

Comment:
Why `anIRI` instead of `iri` (throughout the file)?

Reply:
done (and all similar files above).